util-precompile/PrngSystemContract.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import "./IPrngSystemContract.sol";
+
+contract PrngSystemContract {
+    address constant PRECOMPILE_ADDRESS = address(0x169);
+
+    function getPseudorandomSeed() external returns (bytes32 randomBytes) {
```

Comment:
I still have an aversion to values labeled 'random' that will always deterministically return the same bytes in the same context.
```suggestion
    function getPseudorandomSeed() external returns (bytes32 seedBytes) {
```

Reply:
Fixed. Thanks !